examples/cell-range-selection/README.md

```diff
@@ -0,0 +1,8 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app) and Rescripts.
+
+
+ <!-- Add codesanbox link once deployed -->
```

Comment:
Before merge pls update this, mentioning so that it is not missed

Reply:
updated the link, codesandbox will be available after merge hopefully.